catalog/queries/lookup/aat.rq

```diff
@@ -0,0 +1,49 @@
+PREFIX aat: <http://vocab.getty.edu/aat/>
```

Comment:
So you created two directories: `queries/lookup` and `queries/search`. Would it be easier for maintainers to see the two queries together per data source? In that case we could have either:

- `catalog/queries/aat/{lookup,search}.rq; or
- `catalog/queries/aat-{lookup,serach}.rq`.

Reply:
As discussed, it's indeed easier for maintainers if we group queries per data source. However, quite a number of lookup queries aren't bound to a specific source, but to several sources (in order to handle their common URI prefixes).